rollup.config.js

```diff
@@ -35,14 +35,16 @@ module.exports = ({
   },
   output: [
     {
-      dir: 'release/dist/cjs',
+      dir: 'dist/cjs',
       format: 'cjs',
       sourcemap: true,
+      exports: 'named'
```

Comment:
This fixes a warning:

```
(!) Entry module "src/lib/Sendbird.jsx" is implicitly using "default" export mode, which means for CommonJS output that its default export is assigned to "module.exports". For many tools, such CommonJS output will not be interchangeable with the original ES module. If this is intended, explicitly set "output.exports" to either "auto" or "default", otherwise you might want to consider changing the signature of "src/lib/Sendbird.jsx" to use named exports only.
```